deb2rpm.pl

```diff
@@ -0,0 +1,185 @@
+      $recommends =~ s/(^ | $)//g;
+      # Recommends isn't supported in our version of rpm - so commented out
+      print $out "#Recommends:     $recommends\n";
+    }
+    if (defined($suggests)) {
+      $suggests =~ s/([^() ]*) +\(([<=>]+) +([^() ]+)\)/$1$2$3/g;
+      $suggests =~ s/ *, */ /g;
+      $suggests =~ s/(^ | $)//g;
+      # Suggests isn't supported in our version of rpm - so commented out
+      print $out "#Suggests:       $suggests\n";
+    }
+    print $out <<EOF;
+
+%include %{rootdir}/build-infra/cw-rpm.spec.inc
+
+%description
+$description
+
+%install
+. %{rootdir}/build-infra/cw-rpm-utils $package %{rootdir} %{buildroot}
+setup_buildroot
+EOF
+    if (-e "debian/$package.install") {
+      print $out "install_to_buildroot < %{rootdir}/debian/$package.install\n";
+    }
+    if (-e "debian/$package.dirs") {
+      print $out "dirs_to_buildroot < %{rootdir}/debian/$package.dirs\n";
+    }
+    if (-e "debian/$package.init.d") {
+      print $out "copy_to_buildroot debian/$package.init.d /etc/init.d $package\n";
+    }
+    if (-e "debian/$package.service") {
+      print $out "copy_to_buildroot debian/$package.service /etc/systemd/system\n";
+    }
+    if (-e "debian/$package.logrotate") {
+      print $out "copy_to_buildroot debian/$package.logrotate /etc/logrotate.d $package\n";
+    }
+    if (-e "debian/$package.preinst") {
+      print " - Check debian/$package.preinst!\n";
+      print $out "echo Check debian/$package.preinst and remove this message! >&2 ; exit 1\n";
+    }
+    if (-e "debian/$package.postinst") {
+      print " - Check debian/$package.postinst!\n";
+      print $out "echo Check debian/$package.postinst and remove this message! >&2 ; exit 1\n";
+    }
+    if (-e "debian/$package.prerm") {
+      print " - Check debian/$package.prerm!\n";
+      print $out "echo Check debian/$package.prerm and remove this message! >&2 ; exit 1\n";
+    }
+    if (-e "debian/$package.postrm") {
+      print " - Check debian/$package.postrm!\n";
+      print $out "echo Check debian/$package.postrm and remove this message! >&2 ; exit 1\n";
+    }
+    if (-e "debian/$package.triggers") {
+      print " - Check debian/$package.triggers!\n";
+      print $out "echo Check debian/$package.triggers and remove this message! >&2 ; exit 1\n";
+    }
+    print $out <<EOF;
+build_files_list > $package.files
+
+%files -f $package.files
+EOF
+    close $out;
+  }
+}
+
+my $build_depends;
+my $package;
+my $description;
+my $architecture;
+my $depends;
+my $recommends;
+my $suggests;
+
+while (my $line = <$in>) {
+  chomp $line;
+  if ($line =~ /^Build-Depends: (.*)$/) {
+    $build_depends = $1;
+  } elsif ($line =~ /^Package: (.*)$/) {
+    if (defined $package) {
+      write_spec($package, $build_depends, $description, $architecture, $depends, $recommends, $suggests);
+      undef $description;
+      undef $architecture;
+      undef $depends;
+      undef $recommends;
+      undef $suggests;
+    }
```

Comment:
More intuitive to make this the last elsif block (as we expect the others to have executed by the time we execute this one).